### PR TITLE
[BUGFIX] Updated composer.json makes Packagist work again

### LINKED ISSUE #385: Packagist Update is not working anymore
Packagist does not update the package anymore due to the following error message:

```The friendsoftypo3/frontend-editing package of which you are a maintainer has
failed to update due to invalid data contained in your composer.json.
Please address this as soon as possible since the package stopped updating.

It is recommended that you use `composer validate` to check for errors when you
change your composer.json.

Below is the full update log which should highlight errors as
"Skipped branch ...":

[Composer\Repository\InvalidRepositoryException]: Some branches contained invalid data and were discarded, it is advised to review the log and fix any issues present in branches

Reading composer.json of friendsoftypo3/frontend-editing (v1.8.2)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.8.2)
Reading composer.json of friendsoftypo3/frontend-editing (v1.8.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.8.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.8.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.8.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.7.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.7.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.6.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.6.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.6.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.6.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.5.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.5.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.6)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.6)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.5)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.5)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.4)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.4)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.3)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.3)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.2)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.2)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.4.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.4.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.3.7)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.3.7)
Reading composer.json of friendsoftypo3/frontend-editing (v1.3.6)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.3.6)
Reading composer.json of friendsoftypo3/frontend-editing (v1.3.5)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.3.5)
Reading composer.json of friendsoftypo3/frontend-editing (v.1.3.4)
Skipped tag v.1.3.4, invalid tag name
Reading composer.json of friendsoftypo3/frontend-editing (v.1.3.3)
Skipped tag v.1.3.3, invalid tag name
Reading composer.json of friendsoftypo3/frontend-editing (v.1.3.2)
Skipped tag v.1.3.2, invalid tag name
Reading composer.json of friendsoftypo3/frontend-editing (v.1.3.1)
Skipped tag v.1.3.1, invalid tag name
Reading composer.json of friendsoftypo3/frontend-editing (v1.3.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.3.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.5)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.5)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.4)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.4)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.3)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.3)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.2)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.2)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.2.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.2.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.4)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.4)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.3)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.3)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.2)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.2)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.0)
Reading composer.json of friendsoftypo3/frontend-editing (v1.1.0-alpha)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.1.0-alpha)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.9)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.9)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.8)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.8)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.7)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.7)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.6)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.6)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.5)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.5)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.4)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.4)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.3)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.3)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.2)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.2)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.1)
Reading composer.json of friendsoftypo3/frontend-editing (v1.0.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v1.0.0)
Reading composer.json of friendsoftypo3/frontend-editing (v0.1.0)
Found cached composer.json of friendsoftypo3/frontend-editing (v0.1.0)
Reading composer.json of friendsoftypo3/frontend-editing (v0.0.1)
Found cached composer.json of friendsoftypo3/frontend-editing (v0.0.1)
Reading composer.json of friendsoftypo3/frontend-editing (ckeditor-link)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-ckeditor-link)
Reading composer.json of friendsoftypo3/frontend-editing (ckeditor_configuration)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-ckeditor_configuration)
Reading composer.json of friendsoftypo3/frontend-editing (collapse-inputs)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-collapse-inputs)
Reading composer.json of friendsoftypo3/frontend-editing (composer_replace)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-composer_replace)
Reading composer.json of friendsoftypo3/frontend-editing (input-placeholders)
Importing branch input-placeholders (dev-input-placeholders)
Skipped branch input-placeholders, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

Reading composer.json of friendsoftypo3/frontend-editing (master)
Importing branch master (dev-master)
Skipped branch master, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

Reading composer.json of friendsoftypo3/frontend-editing (no-changes)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-no-changes)
Reading composer.json of friendsoftypo3/frontend-editing (raise_version)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-raise_version)
Reading composer.json of friendsoftypo3/frontend-editing (remove-349-notice-from-documentation)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-remove-349-notice-from-documentation)
Reading composer.json of friendsoftypo3/frontend-editing (typo3_x_compatible)
Importing branch typo3_x_compatible (dev-typo3_x_compatible)
Skipped branch typo3_x_compatible, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

Reading composer.json of friendsoftypo3/frontend-editing (update-version-number-to-1-6-1)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-update-version-number-to-1-6-1)
Reading composer.json of friendsoftypo3/frontend-editing (1.9-features)
Importing branch 1.9-features (dev-1.9-features)
Skipped branch 1.9-features, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

Reading composer.json of friendsoftypo3/frontend-editing (7.6-compatibility)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-7.6-compatibility)
Reading composer.json of friendsoftypo3/frontend-editing (254-ce-no-more-draggable)
Importing branch 254-ce-no-more-draggable (dev-254-ce-no-more-draggable)
Skipped branch 254-ce-no-more-draggable, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

Reading composer.json of friendsoftypo3/frontend-editing (349-editing-panel-is-gone-in-TYPO3-v95)
Found cached composer.json of friendsoftypo3/frontend-editing (dev-349-editing-panel-is-gone-in-TYPO3-v95)
Reading composer.json of friendsoftypo3/frontend-editing (351-review-tests-for-typo3-v9-lts)
Importing branch 351-review-tests-for-typo3-v9-lts (dev-351-review-tests-for-typo3-v9-lts)
Skipped branch 351-review-tests-for-typo3-v9-lts, Invalid package information: 
Deprecation warning: replace.frontend_editing is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9]([_.-]?[a-z0-9]+)*". Make sure you fix this as Composer 2.0 will error.

```

### PR DESCRIPTION
**Pull request by mabolek**
Fixes #385

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
 		"typo3/cms-feedit": "*"
 	},
 	"replace": {
-		"frontend_editing": "self.version",
+		"friendsoftypo3/frontend_editing": "self.version",
```
Comment (bmack) [inline composer.json]: no, just remove that line! otherwise you replace your own version of the same package.

File: composer.json [hunk 2]
```diff
@@ -54,6 +54,7 @@
 			"dev-master": "0.0.x-dev"
 		},
 		"typo3/cms": {
+      "extension-key": "frontend_editing",
```
Comment (bmack) [inline composer.json]: wrong indention

File: composer.json (hunk 1 above)
Comment (mabolek) [reply composer.json]: Thanks. Should be fixed in next commit.

File: composer.json (hunk 2 above)
Comment (mabolek) [reply composer.json]: Whoopsie. Where did that come from?

**Pull request merged**